.github/workflows/documentation.yml

```
@@ -2,8 +2,7 @@ name: Documentation
 
 on:
   push:
-    tags:
-      - v*
+    branches: [main]
```

Setting the publish to the `main` tag will publish with every PR merge right? If that is the case the published version of the package and the docs will be out of date with each other.

I'm not sure if this will fix the branch protection error but maybe?

```suggestion
on:
  release:
    types:
    - published
```

---

I think we should try indeed!